browser/_templates/component-const/new/reactComponent.ejs.t

```diff
@@ -0,0 +1,22 @@
+---
+to: src/components/<%= name %>/<%= name %>.js
+---
+
+import React from 'react';
+import "./<%= name %>.css";
+
+// Function based React Component
+const <%= name %> = (props) => {
```

Comment:
Is there a way to uppercase the component names? i.e. name the component `Button` if I run `hygen component-const new button`.

Reply:
When you use Hygen (as the script is currently configured), in order to upper case the component names, you must do it when you write it into command line - EG: 
`hygen component-const new --name Button`
--> * src/Component/Button/Button.js*

I had intially set it up where it would capitalize, but if the variable name is more than one word, it will only capitalize the first character in the string,  EG: 
`hygen component-const new --name newbutton`
--> * src/Component/Newbutton/Newbutton.js*
so I decided against using it. Would you like me to change it?